container-storage-setup.sh

```diff
@@ -1237,6 +1255,11 @@ if [ -e ${_STORAGE_IN_FILE} ]; then
   source ${_STORAGE_IN_FILE}
 fi
 
+# Error out if CONFIG_NAME is missing.
```

Comment:
I put it in `check_storage_options()` first. However it partitions the disk and add it to the VG group before it errors out. I felt we should error out right away if the `CONFIG_NAME` is missing.